src/TestUtils/src/DeviceTests/AssertionExtensions.Windows.cs

```diff
@@ -231,6 +231,12 @@ public static async Task<T> AttachAndRun<T>(this FrameworkElement view, Func<Win
 
 				try
 				{
+					// Some views (like the switch) are animating when the window is shown
+					// so we need to wait a few milliseconds for the animation to complete.
+					// If the animated views are still a bit flakey, we can increase this to
+					// about 250. If we start going above that then there is likely another
+					// issue.
+					await Task.Delay(100);
```

Comment:
This is the main fix for a few "flaky tests" that are testing controls that make use of animations. Typically animations only start once the control is on-screen, so we need to wiat for them to actually complete befoer snapshots/testing colros/testing state